src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useMemo, useState } from "react";
-import { Link, Paragraph } from "../../components";
+import { Link, Paragraph } from "../../../components";
```

Comment:
I moved this component to the welcome directory (with all the other components which related to the welcome page)